draft-ietf-taps-interface.md

```diff
@@ -737,20 +737,60 @@ early binding when required, for example with some Network Address Translator
 
 ### Using Multicast Endpoints
 
-Specifying a multicast group address on a Local Endpoint will indicate to the Transport
-Services system that the resulting connection will be used to receive multicast messages. The
-Remote Endpoint can be used to filter incoming multicast from specific senders. Such
-a Preconnection will only support calling Listen(), not Initiate(). Calling Listen()
-will cause the Transport Services system to register for receiving multicast, such
-as issuing an IGMP join {{?RFC3376}} or using MLD for IPV6 {{?RFC4604}}. Any Connections
-that are accepted from this Listener are receive-only.
-
-Similarly, specifying a multicast group address on the Remote Endpoint will indicate that the
-resulting connection will be used to send multicast messages, and that the Preconnection will
-support Initiate() but not Listen(). Any Connections created this way are send-only.
-
-A Rendezvous() call on Preconnections containing group addresses results in an
-EstablishmentError as described in {{rendezvous}}.
+To use multicast, a Preconnection is first created with the Local/Remote Endpoint
+specifying the any-source multicast (ASM) or source-specific multicast (SSM) multicast group and destination port number.
+
+Calling Initiate() on that Preconnection creates a Connection that can be
+used to send messages to the multicast group. The Connection object that is
+created will support Send() but not Receive(). Any Connections created this
+way are send-only, and do not join the multicast group. The resulting
+Connection will have a Local Endpoint indicating the local interface to
+which the connection is bound and a Remote Endpoint indicating the
+multicast group.
+
```

Comment:
add "The following API calls can be used to configure a `Preconnection` before calling `Initiate()`: